game_eggs/steamcmd_servers/counter_strike/counter_strike_2/egg-counter--strike2.json

```diff
@@ -0,0 +1,124 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-02-03T01:51:08+01:00",
+    "name": "Counter-Strike 2",
+    "author": "admin@ballaual.de",
+    "description": "For over two decades, Counter-Strike has offered an elite competitive experience, one shaped by millions of players from across the globe. And now the next chapter in the CS story is about to begin. This is Counter-Strike 2.",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/steamcmd:sniper": "ghcr.io\/parkervcp\/steamcmd:sniper"
+    },
+    "file_denylist": [],
+    "startup": ".\/game\/bin\/linuxsteamrt64\/cs2 -dedicated $( [ \"$VAC_ENABLED\" == \"1\" ] || printf %s ' -insecure' ) -ip 0.0.0.0 -port {{SERVER_PORT}} -tv_port {{TV_PORT}} -maxplayers {{MAX_PLAYERS}} $( [ \"$RCON_ENABLED\" == \"0\" ] || printf %s ' -usercon' ) +net_public_adr {{SERVER_IP}} +game_alias {{GAMEMODE}} +map {{SRCDS_MAP}} +sv_setsteamaccount {{STEAM_GSLT}",
```

Comment:
Arround the token a starting " is missing

Reply:
thre is no " at all around the token variable as its not needed. The " at the end comes from the startup line itself